packages/synapse-react-client/src/components/widgets/query-filter/CombinedRangeFacetFilter.test.tsx

```diff
@@ -33,6 +43,28 @@ jest.mock('../RangeSlider/RangeSlider', () => ({
   }),
 }))
 
+const mockQueryResponseData: QueryResultBundle = {
```

Comment:
Mock query response when asking for facet stats (with no selected facets, to ensure full column ranges)